templates/lobbys/lobby.html

```diff
@@ -199,9 +237,13 @@ <h3> Prompts </h3>
                         <td>[[prompt.end]]</td>
 
                         <td><a v-bind:href="'/lobby/' + lobbyInfo.lobby_id + '/prompt/' + prompt.prompt_id">Results</a></td>
+                        <td v-if="editPrompts">
+                            <input type="checkbox" @click="selectPrompt(prompt.prompt_id)">
```

Comment:
Rather than using on-click to manually maintain this state, v-model might be a bit cleaner